app/src/main/java/com/kamron/pogoiv/activities/OcrCalibrationResultActivity.java

```diff
@@ -143,14 +147,21 @@ protected void onCreate(Bundle savedInstanceState) {
         saveCalibrationButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                saveCalibrationButton.setVisibility(View.GONE);
-                backToGoivButton.setVisibility(View.VISIBLE);
                 if (results != null && results.isCompleteCalibration()) {
                     GoIVSettings settings = GoIVSettings.getInstance(OcrCalibrationResultActivity.this);
                     settings.saveScreenCalibrationResults(results);
                     Toast.makeText(OcrCalibrationResultActivity.this,
                             R.string.ocr_calibration_saved, Toast.LENGTH_LONG).show();
                 }
+                Intent intent = new Intent(getOuter(), Pokefly.class);
+                intent.setAction(Pokefly.ACTION_STOP);
```

Comment:
To avoid code duplication don't create the Intent here but use `Pokefly.createStopIntent(Context)`